pkg/harvester/config/harvester.js

```diff
@@ -253,7 +252,7 @@ export function init($plugin, store) {
 
   // singleVirtualCluster
   if (isSingleVirtualCluster) {
-    headers(NAMESPACE, [STATE, NAME_UNLINKED, NS_SNAPSHOT_QUOTA, AGE]);
+    headers(NAMESPACE, [STATE, NAME_UNLINKED, AGE]);
```

Comment:
Conditionaly add NS_SNAPSHOT_QUOTA headers in shell/components/ExplorerProjectsNamespaces.vue and shell/list/namespace.vue